lms/static/lms/js/require-config.js

```diff
@@ -94,7 +94,7 @@
             "catch": "js/vendor/ova/catch/js/catch",
             "handlebars": "js/vendor/ova/catch/js/handlebars-1.1.2",
             "tinymce": "js/vendor/tinymce/js/tinymce/tinymce.full.min",
-            "jquery.tinymce": "js/vendor/tinymce/js/tinymce/jquery.tinymce.min"
+            "jquery.tinymce": "js/vendor/tinymce/js/tinymce/jquery.tinymce.min",
```

Comment:
We don't add trailing commas in JavaScript (you should get a JSHint warning about this).